gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-http-get/src/main/resources/schemas/schema-form.json

```diff
@@ -1,31 +1,31 @@
 {
-  "type": "object",
-  "id": "urn:jsonschema:io:gravitee:plugin:entrypoint:http:get:configuration:HttpGetEntrypointConnectorConfiguration",
-  "properties": {
-    "messagesLimitCount": {
-      "type": "integer",
-      "title": "Limit messages count",
-      "description": "Maximum number of messages to retrieve. Default is 500.",
-      "default": 500
+    "type": "object",
+    "id": "urn:jsonschema:io:gravitee:plugin:entrypoint:http:get:configuration:HttpGetEntrypointConnectorConfiguration",
+    "properties": {
+        "messagesLimitCount": {
+            "type": "integer",
+            "title": "Limit messages count",
+            "description": "Maximum number of messages to retrieve. Default is 500.",
+            "default": 500
+        },
+        "messagesLimitDurationMs": {
+            "type": "number",
+            "title": "Limit messages duration (in ms)",
+            "description": "Maximum duration in milliseconds to wait to retrieve messages. Default is 5000.",
```

Comment:
We could explicitly indicate that, the gateway will wait this maximum time for getting the expected number of message (default 500 messages). The number of messages received by the client may be less than 500 if the delay expires.